src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt

```diff
@@ -25,13 +28,19 @@ class TierCalculationEventListener(
   }
 
   private fun readMessage(wrapper: String?): CalculationEventData {
-    val (message) = objectMapper.readValue(wrapper, Message::class.java)
-    return objectMapper.readValue(message, CalculationEventData::class.java)
+    val message = objectMapper.readValue(wrapper, SQSMessage::class.java)
+    val queueId = hmppsQueueService.findByQueueName("tiercalcqueue")?.id
+    log.info("Received message from {} with messageId:{}", queueId, message?.messageId)
```

Comment:
Using embedded/template expressions in strings in kotlin can make it more readable e.g.
`log.info("Received message from $queueId with messageId: ${message?.messageId}"`

Reply:
done